logs/parse.go

```diff
@@ -3,19 +3,49 @@ package logs
 import (
 	"errors"
 	"fmt"
+	"io"
 	"regexp"
 	"strconv"
 	"strings"
 	"time"
 	"unicode/utf8"
 
+	"github.com/google/uuid"
+
 	"github.com/pganalyze/collector/output/pganalyze_collector"
 	"github.com/pganalyze/collector/state"
 )
 
+const LogPrefixAmazonRds string = "%t:%r:%u@%d:[%p]:"
+const LogPrefixAzure string = "%t-%c-"
+const LogPrefixCustom1 string = "%m [%p][%v] : [%l-1] %q[app=%a] "
+const LogPrefixCustom2 string = "%t [%p-%l] %q%u@%d "
+const LogPrefixCustom3 string = "%m [%p] %q[user=%u,db=%d,app=%a] "
+const LogPrefixCustom4 string = "%m [%p] %q[user=%u,db=%d,app=%a,host=%h] "
+const LogPrefixCustom5 string = "%t [%p]: [%l-1] user=%u,db=%d - PG-%e "
+const LogPrefixCustom6 string = "%t [%p]: [%l-1] user=%u,db=%d,app=%a,client=%h "
+const LogPrefixCustom7 string = "%t [%p]: [%l-1] [trx_id=%x] user=%u,db=%d "
+const LogPrefixCustom8 string = "[%p]: [%l-1] db=%d,user=%u "
+const LogPrefixCustom9 string = "%m %r %u %a [%c] [%p] "
+const LogPrefixCustom10 string = "%m [%p]: [%l-1] db=%d,user=%u "
+const LogPrefixCustom11 string = "pid=%p,user=%u,db=%d,app=%a,client=%h "
+const LogPrefixCustom12 string = "user=%u,db=%d,app=%a,client=%h "
+const LogPrefixCustom13 string = "%p-%s-%c-%l-%h-%u-%d-%m "
+const LogPrefixCustom14 string = "%m [%p][%b][%v][%x] %q[user=%u,db=%d,app=%a] "
+const LogPrefixCustom15 string = "%m [%p] %q%u@%d "
+const LogPrefixCustom16 string = "%t [%p] %q%u@%d %h "
+const LogPrefixSimple string = "%m [%p] "
+const LogPrefixHeroku1 string = " sql_error_code = %e "
+const LogPrefixHeroku2 string = ` sql_error_code = %e time_ms = "%m" pid="%p" proc_start_time="%s" session_id="%c" vtid="%v" tid="%x" log_line="%l" %qdatabase="%d" connection_source="%r" user="%u" application_name="%a" `
```

Comment:
Copied from `legacy_parse.go`. A few places reference some of these, and it's probably good to keep them for reference.